src/app/public/modules/tabs/tab-button.component.ts

```diff
@@ -76,23 +80,19 @@ export class SkyTabButtonComponent {
   @Output()
   public closeClick = new EventEmitter<any>();
 
-  public get queryParams(): {[_: string]: string} {
+  public get tabHref(): string {
```

Comment:
Technically this is a break change as the `queryParams` getter was public and we are removing it. I think it is ok since it isn't an event or input and is something we wouldn't anticipate that consumers would be using but I wanted to double check that we are ok with doing it.

Reply:
Good observation. In the case of components, only `Input` and `Output` properties should be considered for breaking changes. Services are different story, however.